x-pack/plugins/observability_solution/slo/public/plugin.ts

```diff
@@ -159,6 +149,16 @@ export class SloPlugin
     const kibanaVersion = this.initContext.env.packageInfo.version;
     const { ruleTypeRegistry, actionTypeRegistry } = pluginsStart.triggersActionsUi;
 
+    pluginsStart.dashboard.registerDashboardPanelPlacementSetting(
```

Comment:
@nickpeihl The SLO feature is only for users with platinum license and above. Can you check for proper license [check](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/slo/public/plugin.ts#L97) here? Let me know if you want me to do the change and push directly to your branch

Reply:
@nickpeihl I was now checking a draft [PR](https://github.com/elastic/kibana/pull/182621/files#diff-79de7717c424115290175c5d8cf7f9e4cd97cde4a204c742d85a3bc40be5cdacR106) by @ThomThomson and I noticed he didn't move the registration to the `start` method. He kept things in the setup method, he just called the register method from the setup contract. 

In his case `pluginsSetup.embeddable.registerReactEmbeddableFactory`. So maybe you want to try doing something similar `pluginsSetup.dashboard.registerDashboardPanelPlacementSetting`?

Reply:
Oops, you're right. I was misunderstanding the lifecycle. `DashboardStart` is available in the plugins' `Setup` lifecycle via the `core.getStartServices` method. Fixed in https://github.com/elastic/kibana/pull/182571/commits/79d8d9c444de800794fa7121b6fa1a39cbe8bbb0.

Reply:
Ok looks good! Let me quickly try it out locally, and then I'll happily approve

Reply:
@nickpeihl I tested it and I verify that all works well!

<img width="1149" alt="Screenshot 2024-05-06 at 21 53 17" src="https://github.com/elastic/kibana/assets/2852703/32bbcbc4-e994-4e8d-9775-0c2e3d24de1c">